spharpy/_deprecation.py

```diff
@@ -1,13 +1,16 @@
-from spharpy.samplings import Coordinates
-from spharpy.samplings import SamplingSphere
+from spharpy.samplings import Coordinates, SamplingSphere
 import pyfar as pf
 
```

Comment:
```suggestion

    from pyfar.classes import warnings as pwarnings
    from warnings import warn
```
Import deprecation warning